### ISSUE TITLE
InvalidSignatureException with athena:StartQueryExecution

### ISSUE THREAD

**Issue by waffletower-reify** [cognitect-labs/aws-api#261]
Attempting to execute an Athena query.  This exception does not occur when we utilize:
```clojure
{com.cognitect.aws/api {:mvn/version "0.8.673"}
...}
```
## Dependencies

```clojure
        com.cognitect.aws/api {:mvn/version "0.8.711"}
        com.cognitect.aws/endpoints {:mvn/version "1.1.12.772"}
        com.cognitect.aws/athena {:mvn/version "869.2.1687.0"}
```

## Description with failing test case

Exception occurs when we utilize different credentials providers. Here we provide valid credentials via envvars and they are discovered by the default provider chain (Exception occurs after provider chain logic is executed).  Again, this snippet succeeds with api "0.8.693".

```clojure
(require '[cognitect.aws.client.api :as aws])

(let [client (aws/client {:api :athena
                          :region "us-east-1"})]
  (aws/invoke
    client
    {:op :StartQueryExecution
     :request {:QueryString "SELECT COUNT(*) FROM not.a_table"
               :WorkGroup "my-workgroup"}}))
```

Returns `:cognitect.aws.error/code "InvalidSignatureException"`

**Comment by scottbale** [comment]
Thanks for this writeup @waffletower-reify. I have been able to reproduce this issue myself.

Meanwhile, one workaround that might be an option for you: you can use the old cognitect-based http-client in your athena client, even with the latest version of aws-api. This issue only seems to happen when the default new Java-based http-client is used. There are instructions [here](https://github.com/cognitect-labs/aws-api/blob/main/README.md#overriding-the-http-client) about overriding the http-client on a per-aws-client basis. It would look something like this:

```clojure
  (require ...
           '[cognitect.aws.http.cognitect :as cognitect])

  (def client (aws/client {:api :athena
                           :region "us-east-1"
                           :http-client (cognitect/create)}))
```

**Comment by scottbale** [comment]
This should be fixed in latest release `0.8.723`
